applications/MeshingApplication/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ if(${USE_TETGEN_NONFREE_TPL} MATCHES ON)
     include_directories(${TETGEN_INCLUDE})
     add_library(tetgen STATIC ${TETGEN_INCLUDE}/tetgen.cxx ${TETGEN_INCLUDE}/predicates.cxx)
     target_compile_definitions(tetgen PRIVATE -DTETLIBRARY)
+    add_definitions(-DUSE_TETGEN_NONFREE_TPL)
```

Comment:
I think this should be confirmed by @roigcarlo.

Reply:
Just for clarification, if I don't add this line, the code just does not compile, because the variable is not transferred to c++.

Reply:
@roigcarlo do you approve this piece of code?

Reply:
As long as is properly warded behind the `USE_TETGEN_NONFREE_TPL` and people understand that activating it voids the BSD licence, fine to me. It should be clearly specified in the new app README.

That being said I always discourage the use of Tetgen...